independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/strategy/CodestartFileStrategyHandler.java

```diff
@@ -48,4 +49,8 @@ default void createDirectories(Path targetPath) throws IOException {
         Files.createDirectories(targetPath.getParent());
     }
 
+    default void writeFile(final Path targetPath, final String content) throws IOException {
+        Files.write(targetPath, content.getBytes(StandardCharsets.UTF_8));
```

Comment:
This won't work for binary files

Reply:
@gastaldi I don't use String for binary files :)

Binary files are directly copied: 
https://github.com/quarkusio/quarkus/blob/master/independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/CodestartResource.java#L145

Reply:
Ah doh, I missed the method signature 😝 

LGTM then